lib/common/common/src/top_k.rs

```diff
@@ -11,7 +11,7 @@ use crate::types::{ScoreType, ScoredPointOffset};
 #[derive(Default)]
 pub struct TopK {
     k: usize,
-    elements: Vec<Reverse<ScoredPointOffset>>,
+    pub elements: Vec<Reverse<ScoredPointOffset>>,
```

Comment:
I would prefer not to expose this.
You can either return reference to a slice in a method, or deconstruct the whole thing into vector if you need ownership

Reply:
Fixed.